src/utils/utils_posix_common.c

```diff
@@ -0,0 +1,36 @@
+/*
+ *
+ * Copyright (C) 2024 Intel Corporation
+ *
+ * Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ */
+
+#include <stdlib.h>
+#include <string.h>
+
+int util_env_var(const char *envvar, char *buffer, size_t buffer_size) {
+    char *value = getenv(envvar);
+    if (!value) {
+        return 0;
+    }
+
+    size_t len = strlen(value) + 1;
+    if (len > buffer_size) {
+        return len;
+    }
+
+    strcpy(buffer, value);
+
```

Comment:
Shouldn't we you "safe" string functions - do coverity complain about those?

Reply:
It's highly recommended.
AFAIK Coverity doesn't complain about those functions, but hardening-check so (Fortify Source functions).
I'm not sure about BDBA.
https://cwe.mitre.org/data/definitions/676.html

Reply:
Done